packages/blockprotocol/cli/commands.js

```diff
@@ -0,0 +1,7 @@
+import * as help from "./commands/help.js";
+import * as publish from "./commands/publish.js";
+
+export const commands = {
```

Comment:
optional:

```suggestion
export const commandLookup = {
```

Plural suggests we are dealing with an array here, which is not true.

Reply:
https://github.com/blockprotocol/blockprotocol/pull/575/commits/bd8dd3436a6d588d763e6211c473151178f9d134